shoop/themes/classic_gray/templates/classic_gray/navigation_basket_partial.jinja

```diff
@@ -1,3 +1,4 @@
+{% if show_prices() %}
```

Comment:
Might be better to wrap this around and the <ul> in?

https://github.com/shoopio/shoop/blob/master/shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/_navigation.jinja#L104-L106